### ISSUE TITLE
ceiling too low

### ISSUE THREAD

**Issue by x3003** [Realm667/WolfenDoom#1452]
In c3m1_b. this piece of ceiling is too low.

![Screenshot_Doom_20240816_085912](https://github.com/user-attachments/assets/bb2e763d-d943-4922-bb21-751074fb4fea)

**Comment by hawkwind3** [comment]
Yes. It is a bit low. But was designed that way. Same in BOA3.1. Not a bug, so closed.

**Comment by x3003** [comment]
Designed maybe, but even the door is higher than the ceiling. Maybe not a bug, but a mistake.

**Comment by hawkwind3** [comment]
@x3003 and @Tormentor667  - I've given this some thought. If both of you are happy with the attached, I'll commit.

[C3M1_B.zip](https://github.com/user-attachments/files/19406984/C3M1_B.zip)
